engine/src/main/java/org/terasology/engine/rendering/primitives/ChunkTessellator.java

```diff
@@ -43,9 +43,7 @@ public ChunkMesh generateMesh(ChunkView chunkView, float scale, int border) {
             for (int z = 0; z < Chunks.SIZE_Z; z++) {
                 for (int y = 0; y < Chunks.SIZE_Y - border * 2; y++) {
                     Block block = chunkView.getBlock(x, y, z);
-                    if (block != null && block.getMeshGenerator() != null) {
```

Comment:
Why is it safe to remove these checks?

Reply:
there is a default definition and I added a precondition where this can't be null for setMeshGenerator